src/Umbraco.Core/Sync/LastSyncedManager.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Umbraco.Cms.Core.Persistence.Repositories;
+using Umbraco.Cms.Core.Scoping;
+
+namespace Umbraco.Cms.Core.Sync;
+
+public class LastSyncedManager : ILastSyncedManager
+{
+    private readonly ILastSyncedRepository  _lastSyncedRepository;
```

Comment:
```suggestion
    private readonly ILastSyncedRepository _lastSyncedRepository;
```

Double space :smile: